<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AglAllWindow</class>
 <widget class="QMainWindow" name="AglAllWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>agl_all</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="aglall_layout_workDir">
      <item>
       <widget class="QLabel" name="aglall_label_workDir">
        <property name="minimumSize">
         <size>
          <width>110</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string>Work directory</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="aglall_lineEdit_workDir">
        <property name="minimumSize">
         <size>
          <width>280</width>
          <height>23</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Path to work directory</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="aglall_pushButton_workDir">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string>Browse</string>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="aglall_layout_input">
      <item>
       <widget class="QLabel" name="aglall_label_input">
        <property name="minimumSize">
         <size>
          <width>110</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string>Input file</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="aglall_lineEdit_input">
        <property name="minimumSize">
         <size>
          <width>280</width>
          <height>23</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Path to input file</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pdb_pushButton_input">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string>Browse</string>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="aglall_layout_output">
      <item>
       <widget class="QLabel" name="aglall_label_output">
        <property name="minimumSize">
         <size>
          <width>110</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string>Mask of output files</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="aglall_lineEdit_output">
        <property name="minimumSize">
         <size>
          <width>280</width>
          <height>23</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Mask of output files</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="aglall_pushButton_output">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string>Browse</string>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="aglall_layout_num">
      <item>
       <widget class="QLabel" name="aglall_label_file">
        <property name="minimumSize">
         <size>
          <width>110</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string>File</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="aglall_spinBox_file">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>23</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Number of file. '0' is all files</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="maximum">
         <number>999</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="aglall_label_num">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string>Number of molecules</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="aglall_spinBox_num">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>23</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Number of molecules in agglomerates. '0' is all agglomerates</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="aglall_layout_buttons">
      <item>
       <widget class="QPushButton" name="aglall_pushButton_clear">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>23</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Clear window</string>
        </property>
        <property name="text">
         <string>Clear</string>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_15">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="aglall_pushButton_start">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Start analysis</string>
        </property>
        <property name="text">
         <string>Start</string>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="aglall_pushButton_close">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>23</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Close window</string>
        </property>
        <property name="text">
         <string>Close</string>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>aglall_lineEdit_workDir</tabstop>
  <tabstop>aglall_pushButton_workDir</tabstop>
  <tabstop>aglall_lineEdit_input</tabstop>
  <tabstop>pdb_pushButton_input</tabstop>
  <tabstop>aglall_lineEdit_output</tabstop>
  <tabstop>aglall_pushButton_output</tabstop>
  <tabstop>aglall_spinBox_file</tabstop>
  <tabstop>aglall_spinBox_num</tabstop>
  <tabstop>aglall_pushButton_clear</tabstop>
  <tabstop>aglall_pushButton_start</tabstop>
  <tabstop>aglall_pushButton_close</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>aglall_pushButton_close</sender>
   <signal>clicked()</signal>
   <receiver>AglAllWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>462</x>
     <y>225</y>
    </hint>
    <hint type="destinationlabel">
     <x>495</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
